libretro/libretro.cpp

```diff
@@ -559,6 +682,20 @@ static void check_variables(CoreParameter &coreParam)
 
    bool isFastForwarding = environ_cb(RETRO_ENVIRONMENT_GET_FASTFORWARDING, &isFastForwarding);
    coreParam.fastForward = isFastForwarding;
+
+   ppsspp_enable_wlan.Update(&g_Config.bEnableWlan);
+   ppsspp_wlan_channel.Update(&g_Config.iWlanAdhocChannel);
+   ppsspp_discord_presence.Update(&g_Config.bDiscordPresence);
+   ppsspp_enable_builtin_pro_ad_hoc_server.Update(&g_Config.bEnableAdhocServer);
+
+   ppsspp_chat_button_position.Update(&g_Config.iChatButtonPosition);
+   ppsspp_chat_screen_position.Update(&g_Config.iChatButtonPosition);
```

Comment:
Should be `iChatScreenPosition`.

Reply:
Good catch. I'm not sure whether the chat options make a lot of sense for devices without keyboard. However reading what other people type could still be good.

Furthermore the chat button would be part of PPSSPP's on-screen keyboard? Can that be used from the Libretro core?

Reply:
The chat functionality has been removed for the Libretro core. It doesn't make sense in the Libretro core since the chat overlay is not displayed by the front end.